### PR TITLE
Removing errors in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by DhruvamSharma**
Files should be grammatical error free.

### CODE REVIEW

**Comment by amarjeetkapoor1** [comment]
It's showing conflict. So, Would you solve that?

**Comment by DhruvamSharma** [comment]
Done, as requested .

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,15 +12,15 @@ For contributing to this repository follow following steps:
...
     * Make sure your PR adds only one functionality or try to fix one issue per PR.
-    * Don't try to do a lot in one PR i.e. stick to scope of your PR.
+    * Don't try to do a lot in one PR i.e. stick to the scope of your PR.
     * Your PR should take code from one stable state to another stable state.
+
+2. For Commits:
+    * Try to have proper commit messages which tell what you commit dos.
     * If your PR solves some issue, refer that issue in your PR.
     * Provide proper description what your PR do.(Description in terms of Points will be plus).
     * If your are still working on PR. Label it (Work in Progress) and Add TODO's in description (How to do that given at end of doc) and keep marking once done.
```
Comment (amarjeetkapoor1) [inline CONTRIBUTING.md]: Would Change do following changes here ?
1. s/you/your/
2. s/description/the description/

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,15 +12,15 @@ For contributing to this repository follow following steps:
...
 1. When submitting PR:
     * Make sure your PR adds only one functionality or try to fix one issue per PR.
-    * Don't try to do a lot in one PR i.e. stick to scope of your PR.
+    * Don't try to do a lot in one PR i.e. stick to the scope of your PR.
     * Your PR should take code from one stable state to another stable state.
+
+2. For Commits:
+    * Try to have proper commit messages which tell what you commit dos.
     * If your PR solves some issue, refer that issue in your PR.
     * Provide proper description what your PR do.(Description in terms of Points will be plus).
```
Comment (amarjeetkapoor1) [inline CONTRIBUTING.md]: change do to does.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -12,14 +12,14 @@ For contributing to this repository follow following steps:
...
-    * Don't try to do a lot in one PR i.e. stick to scope of your PR.
+    * Don't try to do a lot in one PR i.e. stick to the scope of your PR.
     * Your PR should take code from one stable state to another stable state.
-    * If your PR solves some issue, refer that issue in your PR.
-    * Provide proper description what your PR do.(Description in terms of Points will be plus).
-    * If your are still working on PR. Label it (Work in Progress) and Add TODO's in description (How to do that given at end of doc) and keep marking once done.
 
 2. For Commits:
-    * Try to have proper commit messages which tells what your commit does.
+    * Try to have proper commit messages which tell what you commit dos.
```
Comment (amarjeetkapoor1) [inline CONTRIBUTING.md]: Shouldn't it be "does" instead of do?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -12,14 +12,14 @@ For contributing to this repository follow following steps:
...
+    * If your PR solves some issue, refer that issue to your PR.
```
Comment (amarjeetkapoor1) [inline CONTRIBUTING.md]: and Isn't "in" more correct here instead of "to"?

**Comment by DhruvamSharma** [comment]
Done!

**Pull request merged**